source/content/object-cache.md

```diff
@@ -202,11 +202,13 @@ TRUNCATE TABLE `<tablename>`;
 
 <Tab title="Drupal 7" id={"d7-install"}>
 
-<Alert title="Note" type="info">
+#### Before You Begin
 
-This configuration uses the `Redis_CacheCompressed` class for better performance. This requires the Redis module version 3.13 or later. For versions before 3.13, use `Redis_Cache` in step 4 instead.
+Please note:
 
-</Alert>
+* This configuration uses the `Redis_CacheCompressed` class for better performance. This requires the Redis module version 3.13 or later. For versions before 3.13, use `Redis_Cache` in step 4 instead.
+
+* The version of Redis for Drupal 7 isn’t compatible with PHP 7.4, as PHP 7.4 uses php-redis 5. We recommend that you upgrade your version of PHP to avoid any errors, or you may consider using [this patch](https://www.drupal.org/project/redis/issues/3074189) as a workaround.
```

Comment:
Hmm, so PHP 7.4 is the recommended version to use and if they do update, they will get errors. So this part really doesn't work. Maybe something like this?

> The version of Redis for Drupal 7 isn’t compatible with PHP 7.4, as PHP 7.4 uses php-redis 5. We recommend that you should use PHP 7.4 as this is the only supported version of PHP 7, but you may consider using [this patch](https://www.drupal.org/project/redis/issues/3074189) to get the Redis module working with PHP 7.4.

In my PR, I also added a bit more info in the FAQ section, just in case customers aren't looking at the "Install" section for an issue after an upgrade: https://github.com/pantheon-systems/documentation/pull/6977

Reply:
Thanks for your review, Dan! I will close this PR, in favor of yours.